tools/make/lib/lint/typescript/eslint.mk

```diff
@@ -0,0 +1,186 @@
+#/
+# @license Apache-2.0
+#
+# Copyright (c) 2017 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#/
+
+# VARIABLES #
+
+# Define the path to the [ESLint][1] executable.
+#
+# To install ESLint:
+#
+# ```bash
+# $ npm install eslint
+# ```
+#
+# [1]: https://eslint.org/
+ESLINT ?= $(BIN_DIR)/eslint
+
+# Define the path to the ESLint configuration file:
+ESLINT_TS_CONF ?= $(CONFIG_DIR)/eslint/.eslintrc.typescript.js
+
+# Define the path to the ESLint ignore file:
+ESLINT_IGNORE ?= $(CONFIG_DIR)/eslint/.eslintignore
+
+# Define the command-line options to use when invoking the ESLint executable:
+ESLINT_TS_FLAGS ?= \
+	--ignore-path $(ESLINT_IGNORE)
+
+ifeq ($(AUTOFIX),true)
+	ESLINT_TS_FLAGS += --fix
+endif
+
+FIX_TYPE ?=
+ifneq ($(FIX_TYPE),)
+	ESLINT_TS_FLAGS += --fix-type $(FIX_TYPE)
+endif
+
+# RULES #
+
+#/
+# Lints TypeScript source files using [ESLint][1].
+#
+# ## Notes
+#
+# -   This rule is useful when wanting to glob for TypeScript source files (e.g., lint all TypeScript source files for a particular package).
+#
+# [1]: https://eslint.org/
+#
+# @private
+# @param {string} [SOURCES_FILTER] - file path pattern (e.g., `.*/math/base/special/abs/.*`)
```

Comment:
Same comment as above.

Reply:
Ah, now I see. I think this is a bug: https://github.com/stdlib-js/stdlib/blob/ae9cdb7f4a5bad86dfb5f67f3ea7656e362a71a7/tools/make/lib/ls/typescript/sources.mk#L25. Instead of `SOURCES_FILTER` here, we should be using `TYPESCRIPT_SOURCES_FILTER`.